experimental/dds/tree2/src/feature-libraries/sequence-field/sequenceFieldToDelta.ts

```diff
@@ -78,16 +86,12 @@ export function sequenceFieldToDelta<TNodeChange>(
 					break;
 				}
 				case "Revive": {
-					if (mark.conflictsWith === undefined) {
-						const insertMark: Mutable<Delta.Insert> = {
-							type: Delta.MarkType.Insert,
-							content: mark.content,
-						};
-						populateChildModificationsIfAny(mark.changes, insertMark, deltaFromChild);
-						out.pushContent(insertMark);
-					} else if (mark.lastDetachedBy === undefined) {
-						out.pushOffset(mark.count);
-					}
+					const insertMark: Mutable<Delta.Insert> = {
```

Comment:
I think we need a check against blocked revives.
Something like `mark.inverseOf !== undefined && mark.inverseOf !== mark.detachEvent?.revision` should take cake of it.

Reply:
We check whether the mark has a cell effect before switching on the type of the mark. A blocked revive would be caught there.